multithread_test.py

```diff
@@ -0,0 +1,82 @@
+# Help from : https://pymotw.com/2/threading/, https://www.tutorialspoint.com/python/python_networking.htm, https://github.com/UCLA-CS130/AAAAA/blob/master/test/integration_multithread_test.py
+
+from os import system
+from threading import Thread
+import socket
+import sys
+import subprocess
+import time
+
+PORT = 2020
+BUFFER_LEN = 1024
+HOLD_REQUEST = "GET /hold HTTP/1.1"
```

Comment:
Technically supposed to have CRLF but also it doesn't actually matter according to the holding handler code huh.